cognitedata/reveal
# fix(react-components): Point cloud layer visibility issue

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-2315

## Description :pencil:
When point cloud `styling` value is passed through `RevealContainer` and if the point cloud model is un-selected in the `Layer` button than when page is refreshed, the un-selected point model is visible as the styling value from the `RevealContianer` overrides the visibility applied when model is loaded.

This PR fixes the problem.

## How has this been tested? :mag:

In Storybook and FDX through yalc

## Test instructions :information_source:

- `cd react-components && yarn && yarn build && yarn storybook`
- Select `Reveal3DResources` example
- Open the canvas of the story in a new window
- Change the 3D resources data by unselecting point-cloud model in `Layer Button` and confirm that the state changes in Url
- Can copy the Url and load it in new tab.

## Checklist :ballot_box_with_check:
